test/language/expressions/assignmenttargettype/direct-callexpression-arguments.js

```diff
@@ -17,7 +17,4 @@ info: |
 
 $DONOTEVALUATE();
 
-function _() {
-  f() = 1;
-}
-
+f() = 1;
```

Comment:
For this + other tests that do `f()`, I think we need to make sure `f` is a function — otherwise Web-compatible engines produce a `ReferenceError` which is not what the test is about.

Adding a line above that `function f() {}` might be sufficient. WDYT?